.gitattributes

```diff
@@ -1 +0,0 @@
-* text=auto
```

Comment:
Why this change? Seems unrelated to typos in help messages

Reply:
Yah that's true.. I don't think that attribute works very reliably across platforms - or at least it does not seem to do anything, all files are still being pushed as CRLF and thus failing checkstyle. I played around with enforcing gitattributes, but its still not really reliable.

Instead, I think [`.editorconfig`](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/master/.editorconfig) is more reliable, which is also understood by GitHub and IDE's.